maven-plugin/src/main/java/com/marvinformatics/formatter/LineEnding.java

```diff
@@ -21,56 +21,56 @@
  */
 public enum LineEnding {
 
-	AUTO(System.getProperty("line.separator")),
-	KEEP(null),
-	LF("\n"),
-	CRLF("\r\n"),
-	CR("\r"),
-	UNKNOW(null);
+  AUTO(System.getProperty("line.separator")),
+  KEEP(null),
+  LF("\n"),
+  CRLF("\r\n"),
+  CR("\r"),
+  UNKNOW(null);
 
-	private final String chars;
+  private final String chars;
 
-	LineEnding(String chars) {
-		this.chars = chars;
-	}
+  LineEnding(String chars) {
+    this.chars = chars;
+  }
 
-	public String getChars() {
-		return chars;
-	}
+  public String getChars() {
+    return chars;
+  }
 
-	/**
-	 * Returns the most occurring line-ending characters in the file text or null if no line-ending occurs the most.
-	 * 
-	 * @return
-	 */
-	public static LineEnding determineLineEnding(String fileDataString) {
-		int lfCount = 0;
-		int crCount = 0;
-		int crlfCount = 0;
+  /**
+   * Returns the most occurring line-ending characters in the file text or null if no line-ending occurs the most.
+   * 
+   * @return
+   */
+  public static LineEnding determineLineEnding(String fileDataString) {
```

Comment:
The Cyclomatic Complexity of this method "determineLineEnding" is 15 which is greater than 10 authorized.